src/services/refactors/convertExport.ts

```diff
@@ -107,7 +113,16 @@ namespace ts.refactor {
 
     function changeExport(exportingSourceFile: SourceFile, { wasDefault, exportNode, exportName }: ExportInfo, changes: textChanges.ChangeTracker, checker: TypeChecker): void {
         if (wasDefault) {
-            changes.delete(exportingSourceFile, Debug.checkDefined(findModifier(exportNode, SyntaxKind.DefaultKeyword), "Should find a default keyword in modifier list"));
+            if (isExportAssignment(exportNode) && !exportNode.isExportEquals) {
+                const defaultKeyword = Debug.checkDefined(findChildOfKind(exportNode, SyntaxKind.DefaultKeyword, exportingSourceFile), "Should find an default keyword in child list");
+                changes.delete(exportingSourceFile, defaultKeyword);
+                const exp = exportNode.expression as Identifier;
+                const spec = makeExportSpecifier(exp.text, exp.text);
+                changes.replaceNode(exportingSourceFile, exp, factory.createNamedExports([spec]));

```

Comment:
It seems like just replacing the whole export assignment node would be simpler than deleting the `default` token and replacing the exported expression.

Reply:
Thanks for advice.